org.pjbroad.EternallandsClient.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It would be better to maintain this file in main repo so everyone can benefit from it.

Reply:
Also the same applies to the png icon.

Reply:
Yes, but as before in a future release